framework/src/test/java/org/tron/core/db/AccountAssetStoreTest.java

```diff
@@ -0,0 +1,139 @@
+package org.tron.core.db;
+
+import com.google.common.primitives.Longs;
+import com.google.protobuf.ByteString;
+
+import java.util.Map;
+import javax.annotation.Resource;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.tron.common.BaseTest;
+import org.tron.common.utils.ByteArray;
+import org.tron.core.Constant;
+import org.tron.core.Wallet;
+import org.tron.core.capsule.AccountCapsule;
+import org.tron.core.capsule.AssetIssueCapsule;
+import org.tron.core.config.args.Args;
+import org.tron.core.store.AccountAssetStore;
+import org.tron.core.store.AccountStore;
+import org.tron.protos.Protocol;
+import org.tron.protos.contract.AssetIssueContractOuterClass;
+
+public class AccountAssetStoreTest extends BaseTest {
+
+  private static final byte[] ASSET_KEY = "20000".getBytes();
+  private static AccountCapsule ownerCapsule;
+
+  private static String OWNER_ADDRESS = Wallet.getAddressPreFixString()
+          + "abd4b9367799eaa3197fecb144eb71de1e049abc";
+  private static final long TOTAL_SUPPLY = 1000_000_000L;
+  private static final int TRX_NUM = 10;
+  private static final int NUM = 1;
+  private static final long START_TIME = 1;
+  private static final long END_TIME = 2;
+  private static final int VOTE_SCORE = 2;
+  private static final String DESCRIPTION = "TRX";
+  private static final String URL = "https://tron.network";
+
+  @Resource
+  private AccountAssetStore accountAssetStore;
+
+  @Resource
+  private AccountStore accountStore;
+
+  static {
+    dbPath = "db_AccountAssetStore_test";
+    Args.setParam(
+            new String[]{
+                "--output-directory", dbPath,
+            },
+            Constant.TEST_CONF
+    );
+  }
+
+  @Before
+  public void init() {
+    accountAssetStore.put(ASSET_KEY, Longs.toByteArray(200L));
+
```

Comment:
No need to add blank line

Reply:
This blank line does not affect the logic.